ui/app/models/pki/tidy.js

```diff
@@ -76,7 +96,7 @@ export default class PkiTidyModel extends Model {
   })
   revocationQueueSafetyBuffer; // enterprise only
 
-  @attr('string', {
```

Comment:
removed 'string' types for ttls so we can pull default values seamlessly from API